website/docs/api/docusaurus.config.js.md

```diff
@@ -174,9 +174,7 @@ module.exports = {
       switchConfig: {
         darkIcon: '🌙',
         lightIcon: '\u2600',
-        // React inline style objects
-        // see https://reactjs.org/docs/dom-elements.html#style
-        darkIconStyle: {
+        darkIconStyle: { // CSS object with camelCased properties
```

Comment:
why not leave my comment? It's not worth to document camelCased etc, React developers know about inline style objects, and non-react devs will read the link.

Reply:
Sorry, my thinking was if we were adding more explanation to the theme classic api page we could reduce the comments in the docusaurus.config.js api doc as I didn't see any other explanatory comments there but you're right, the previous version was better.